client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -130,20 +137,25 @@ def _predict(
             gzstream.seek(0)
 
             response = self._session.post(
-                self._prediction_url,
+                prediction_url,
                 headers=request_headers,
                 data=gzstream.read(),
-                )
+            )
         else:
+            # when passing json=x, requests sets `allow_nan=False` by default (as of 2.26.0), which we don't want
+            # so we're going to dump ourselves
+            body = json.dumps(x, allow_nan=True)
+            if not isinstance(body, bytes):
+                body = body.encode("utf-8")
```

Comment:
We can omit this since we're firmly in Python 3, now. `json.dumps()` will give us a `utf-8` `str`